container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_api_container_functions.go

```diff
@@ -148,15 +147,7 @@ func (backend *DockerKurtosisBackend) CreateAPIContainer(
 		labelStrs[labelKey.GetString()] = labelValue.GetString()
 	}
 
-	//The APIContainer will be configured to send the logs to the Fluentbit logs collector server
-	logCollectorAddr, err := enclaveLogsCollector.GetEnclaveNetworkAddressString()
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "An error occurred retrieving the log collector address.")
-	}
-	fluentdLoggingDriverCnfg := docker_manager.NewFluentdLoggingDriver(
-		logCollectorAddr,
-		logs_collector_functions.GetKurtosisTrackedLogsCollectorLabels(),
-	)
+	// TODO: configure the APIContainer to send the logs to the Fluentbit logs collector server
```

Comment:
For now, I've stopped the APIC from sending logs to the logs aggregator. This requires extra configuration to vector that's out of scope for this PR. I will add this back in along with configuring the engine to send logs to the logs aggregator.